segment_anything.py

```diff
@@ -145,10 +146,13 @@ def apply(self, image, is_point):
         API_KEY = get_api_key()
         SIZE_LIMIT = 1536
 
-        image, _ = LoadImage().load_image(image)
+        # 加载原始图像
+        original_image, _ = LoadImage().load_image(image)
+        # 直接克隆原始图像用于处理和透传
+        image_to_process = original_image.clone()
```

Comment:
```suggestion
        image_to_process, _ = LoadImage().load_image(image)
```

original_image 并未直接在后续代码中使用。 这里可以不需要 clone 避免多余复制。

Reply:
受教了！

Reply:
对的，担心的就是2这种节点影响了3这种节点，看来这种情况也是无法通过我们编写的1号节点来规避，只有2号节点做好规范才行。